static/js/components/search/LearnerResult_test.js

```diff
@@ -2,20 +2,41 @@
 /* global SETTINGS: false */
 import React from 'react';
 import R from 'ramda';
+import { Provider } from 'react-redux';
 import { assert } from 'chai';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
 
 import LearnerResult from './LearnerResult';
+import { configureMainTestStore } from '../../store/configureStore';
 import { makeStrippedHtml } from '../../util/util';
 import {
   USER_PROFILE_RESPONSE,
   USER_PROGRAM_RESPONSE
 } from '../../constants';
+import rootReducer from '../../reducers';
+import { localStorageMock } from '../../util/test_utils';
 
 describe('LearnerResult', () => {
+  let store;
+
   let renderLearnerResult = props => (
-    makeStrippedHtml(<LearnerResult {...props} />)
+    makeStrippedHtml(
+      <MuiThemeProvider muiTheme={getMuiTheme()}>
+        <Provider store={store}>
+          <LearnerResult {...props} />
+        </Provider>
+      </MuiThemeProvider>
+    )
   );
 
+  beforeEach(() => {
```

Comment:
the changes here are necessary because `<LearnerResult>` includes a `<ProfileImage>`, which is now a container and not simply a presentational component. So I had to add stuff to this file to avoid errors because there wasn't any store for `<ProfileImage>` to hook into.